src/spec-node/singleContainer.ts

```diff
@@ -202,6 +202,9 @@ async function buildAndExtendImage(buildParams: DockerResolverParameters, config
 		if (buildParams.buildxCacheTo) {
 			args.push('--cache-to', buildParams.buildxCacheTo);
 		}
+		if (buildParams.additionalCacheFroms) {
+			buildParams.additionalCacheFroms.forEach(cacheFrom => args.push('--cache-from', cacheFrom));
```

Comment:
Line 231 below already does this. 🤔

Reply:
hmm, I don't remember why I added it, sorry. It also seems to be working without. 🤷